### PR TITLE
added containers for appropriate links margin size, added links styli…

### PR DESCRIPTION
**Pull request by milklot**
Bottom Nav Bar: Fix Links FIXED

File path : src/components/pages/GamePlay/Footer.js
Put links in the container for the correct margin purpose.

File path: src/styles/less/GamemodeStyles.less
Changed color of links and added underline styling as on a Trello card.
Added hover effect.

The redirection pages for those link are no exist yet. I left them blank for now. In the App there'r some pages exist , but on a parent side. Not sure if child should have an access/redirection on a parent side of App.

Trello card :
https://trello.com/c/eLbyLM41/660-bottom-nav-bar-fix-links

Let me know if you have any suggestions / recommendations for it, thanks

### CODE REVIEW

**Review by Veilios — DISMISSED** [review]
Everything looks clean and easy to follow! Nice work!

File: src/components/pages/GamePlay/Footer.js [hunk 1]
```diff
@@ -10,11 +10,27 @@ export default function Footer() {
...
-          <Link to="#">Terms</Link>
+        <nav className="footer-nav-container">
+          <div className="footer-nav-div-link">
+            <Link className="footer-nav-link" to="#">
+              Contact
+            </Link>
+          </div>
+          <div className="footer-nav-div-link">
+            <Link className="footer-nav-link" to="#">
+              FaQs
```
Comment (rhiannonstanford) [inline src/components/pages/GamePlay/Footer.js]: Please change this to "FAQ"

**Review by rhiannonstanford — CHANGES_REQUESTED** [review]
Great job!  Please see comments.  Just change to "FAQ"   Thank you!

File: src/components/pages/GamePlay/Footer.js (hunk 1 above)
Comment (milklot) [reply src/components/pages/GamePlay/Footer.js]: Done

**Review by rhiannonstanford — APPROVED** [review]
Thank you for that change!  Looks great!

**Review by mollybee — APPROVED** [review]
Container naming looks good and descriptive

**Pull request merged**